client/src/components/Profile.js

```diff
@@ -22,9 +23,7 @@ export default class Profile extends Component {
         const isViewOnly = this.props.isViewOnly;
 
         const linkedInUpdate = (
-            <Button primary>
-                Update info with LinkedIn
-            </Button>
+            <LinkedInUpdate/>
```

Comment:
TODO: email should be passed in as prop here if we do in fact plan to use this method of fetching profile pics

Reply:
Another option would be to just pass in the profile data to this component and store it as a state - we would still need to have isViewOnly